React/Base/RCTUIKit.h

```diff
@@ -391,7 +391,6 @@ CGPathRef UIBezierPathCreateCGPathRef(UIBezierPath *path);
 - (void)layoutIfNeeded;
 
 - (void)layoutSubviews;
-- (NSArray<RCTUIView *> *)reactZIndexSortedSubviews; // [macOS]
```

Comment:
Why is this removed?

Reply:
This was added in 0.68 where RCTUIView did not extend RCTPlatformView. This meant we were not able to use the existing hit test implementation and we needed to re-use the paper hit test implementation that depended on that method.

On fabric, views are always inserted in the right z-index order, so we don't need to re-order views before hit testing.